Clockify.Net/ClockifyClient.Users.cs

```diff
@@ -25,9 +25,12 @@ public async Task<Response<List<UserDto>>> FindAllUsersOnWorkspaceAsync(string w
         /// <summary>
         /// Get currently logged in user's info
         /// </summary>
-        public async Task<Response<CurrentUserDto>> GetCurrentUserAsync()
+        public async Task<Response<CurrentUserDto>> GetCurrentUserAsync(bool? includeMemberships = null)
         {
             var request = new RestRequest("user");
+
+            if (includeMemberships != null) { request.AddQueryParameter("include-memberships", includeMemberships.ToString()); };
```

Comment:
This seems to be better suited for a `bool includeMemberships = false`.

on the API side of things excluding the parameter is the same as send `include-memberships=false` so from a design perspective it might make more sense to just let it be false by default. This should be more clear for intent to consuming developers as well.
```suggestion
        public async Task<Response<CurrentUserDto>> GetCurrentUserAsync(bool includeMemberships = false)
        {
            var request = new RestRequest("user");

            if (includeMemberships) 
            {
                 request.AddQueryParameter("include-memberships", true) ;
            };
```

Reply:
I agree.